### PR TITLE
(@W-8946095) Update the groupId and version, add .strata.yml

### PR DESCRIPTION
**Pull request by sruizEG**
To allow for deployment on Nexus for Falcon integration.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.mayconbordin</groupId>
+    <groupId>com.evergage.thirdparty</groupId>
     <artifactId>streaminer</artifactId>
-    <version>1.2-evg2</version>
+    <version>1.2-evg3</version>
```
Comment (ispringer) [inline pom.xml]: Is this the one where you're changing the version because you accidentally deployed a jdk11-built jar?

Comment (sruizEG) [reply pom.xml]: No sir, it just seemed to make sense to update the version string when deploying to a new coordinate location, as I've updated such strings in other repos; I can revert that line if this simple shift doesn't call for a version number change, no problem.

Comment (ispringer) [reply pom.xml]: I think it's a little less confusing to keep the version the same, if we're not making any actual code changes.

Comment (sruizEG) [reply pom.xml]: @ispringer Alright, got that updated in both this PR and the related `evergage-product` PR at https://github.com/evergage/evergage-product/pull/2141

**Review by ispringer — APPROVED** [review]

**Pull request merged**